### PR TITLE
Create Executive vote - July 12, 2023.md

### PR DESCRIPTION
**Pull request by patrick-j-govalpha**

### CODE REVIEW

File: governance/votes/Executive vote - July 12, 2023.md [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+The Governance Facilitators and Sidestream, PullUp, and dewiz have placed an executive proposal into the voting system. MKR Holders should vote for this proposal if they support the following alterations to the Maker Protocol.
+
+If you are new to voting in the Maker Protocol, please see the [voting guide](https://manual.makerdao.com/governance/voting-in-makerdao/on-chain-governance) to learn how voting works.
+
+---
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes and additions** will occur within the Maker Protocol:
+- A new multiswap output conduit for BlockTower Andromeda (RWA015-A) will be deloyed, as detailed below.
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:19]:
```suggestion
- A new multiswap output conduit for BlockTower Andromeda (RWA015-A) will be deployed, as detailed below.
```

File: governance/votes/Executive vote - July 12, 2023.md [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+
+---
+
+## Proposal Details
+
+### BlockTower Andromeda (RWA015-A) Update
+
+As per the original [technical assessment](https://forum.makerdao.com/t/rwa015-project-andromeda-technical-assessment/20974) a multiswap Output Conduit was initially proposed for this vault type. This will allow the `operator` role to choose which PSM is used to obtain redeemable stablecoins in return for the DAI drawn from the vault. At present the vault is set up to only use the USDP PSM as a compromise while the multiswap tool was developed. The multiswap Output Conduit is now audited and deployed, and the following actions will be carried out if this executive proposal passes.
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:42]: DAI vs Dai throughout exec

File: governance/votes/Executive vote - July 12, 2023.md [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+
+- A new Output Conduit at [0x1E86CB085f249772f7e7443631a87c6BDba2aCEb](https://etherscan.io/address/0x1E86CB085f249772f7e7443631a87c6BDba2aCEb) will be authorized, allowing the `operator` to select which PSM BlockTower Andromeda transactions are routed through. Operationally, this means that USDC, GUSD, or USDP may be used for drawdowns.
+- The relevant [Chainlog](https://chainlog.makerdao.com/) entry `RWA015_A_OUTPUT_CONDUIT` will be updated to reflect the change to the Output Conduit.
+- The USDP, GUSD, and USDC PSMs will be whitelisted for use by calling `clap`.
+- The current Output Conduit will be deauthorized.
+- The old `RWA015_A_OUTPUT_CONDUIT_LEGACY` will be removed from the chainlog as it will no longer be used.
+
+### Smart Burn Engine Deployment
+
+As per this successful govenrance [poll](https://vote.makerdao.com/polling/QmQmxEZp) and the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#9-surplus-buffer-and-smart-burn-engine) a new Smart Burn Engine will be deployed along with the following parameters, if this executive proposal passes.
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:52]:
```suggestion
As per this successful governance [poll](https://vote.makerdao.com/polling/QmQmxEZp) and the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104#9-surplus-buffer-and-smart-burn-engine) a new Smart Burn Engine will be deployed along with the following parameters, if this executive proposal passes.
```

File: governance/votes/Executive vote - July 12, 2023.md [hunk 4]
```diff
@@ -0,0 +1,202 @@
...
+Per the [Support Scope](https://mips.makerdao.com/mips/details/MIP106#7-4-currently-incubating) and the [Accessibility Scope](https://mips.makerdao.com/mips/details/MIP108) the following MKR Budget Streams will be set up, if this executive proposal passes.
+
+| Destination                   | Start Date | End Date   | Cliff Date | Address                                    | Amount (MKR) |
+|-------------------------------|------------|------------|------------|--------------------------------------------|--------------|
+| Chronicle Labs Auditor Wallet | 2023-07-01 | 2024-06-30 | 2023-07-01 | [0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f](https://etherscan.io/address/0x68D0ca2d5Ac777F6A9b0d1be44332BB3d5981C2f) | 2,216.4      |
+| Jetstream Auditor Wallet      | 2023-06-26 | 2024-12-31 | 2023-06-26 | [0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962](https://etherscan.io/address/0xF478A08C41ad06E8D957d5e6B6Bcde7452cEE962) | 1,619.93     |
+
+#### Ecosystem Actor DAI Transfer
+
+As part of the Launch Project the following Dai transfer will be made if this executive proposal passes.
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:162]:
```suggestion
As part of the Launch Project, the following Dai transfer will be made if this executive proposal passes.
```

File: governance/votes/Executive vote - July 12, 2023.md [hunk 5]
```diff
@@ -0,0 +1,202 @@
...
+
+#### DUX-001
+
+As part of their [voluntary offboarding](https://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306), **56.48 MKR** will be transferred to the DUX Core Unit Auditor Wallet at [0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad](https://etherscan.io/address/0x5A994D8428CCEbCC153863CCdA9D2Be6352f89ad) if this executive proposal passes.
+
+This is the remainder of the MKR due to the DUX Core Unit under that agreement. The original MKR budget may be seen [here](https://mips.makerdao.com/mips/details/MIP40c3SP27).
+
+### Cancel Core Unit DAI stream
+
+As part of their As part of their [voluntary offboarding](https://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306), DAI stream ID 14 to DUX-001 will be cancelled, if this executive proposal passes.
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:182]:
```suggestion
As part of their [voluntary offboarding](https://forum.makerdao.com/t/mip39c3-sp9-removing-dux-001/21306), DAI stream ID 14 to DUX-001 will be cancelled, if this executive proposal passes.
```

File: governance/votes/Executive vote - July 12, 2023.md [hunk 6]
```diff
@@ -0,0 +1,202 @@
+---
+title: Template - [Executive Vote] BlockTower Andromeda Upgrade, Smart Burn Engine Deployment, Keeper Job Updates, Scope Defined Parameter Changes, Delegate Compensation, Ecosystem Actor Funding, Spark Protocol Proxy Spell Execution - July 12, 2023
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:2]: Seems to be missing the CRVV1ETHSTETH-A 1st Stage Offboarding. Please confirm if it's not meant for this exec. Would also reword to "Ecosystem Actor and Core Unit Funding Updates" to catch the yanked streams and MKR vesting to CUs. Applies to summary below.

File: governance/votes/Executive vote - July 12, 2023.md [hunk 7]
```diff
@@ -0,0 +1,202 @@
...
+
+## Executive Summary
+
+If this executive proposal passes, the following **changes and additions** will occur within the Maker Protocol:
+- A new multiswap output conduit for BlockTower Andromeda (RWA015-A) will be deloyed, as detailed below.
+- The new Smart Burn Engine will be deployed, as detailed below.
+- Keeper jobs will be updated, as detailed below.
+- Multiple parameter changes per the Stability Scope will be made, as detailed below.
+- Delegate compensation for June 2023 will be distributed, as detailed below.
+- Funding stream setup and housekeeping for two Ecosystem Actors will be carried out, as detailed below.
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:24]: CUs here too, if changed above

File: governance/votes/Executive vote - July 12, 2023.md [hunk 8]
```diff
@@ -0,0 +1,202 @@
...
+- The [Surplus Auction Lot Size (`vow.bump`)](https://manual.makerdao.com/parameter-index/surplus-auction/param-surplus-lot-size) will be reduced by 25,000 DAI from 30,000 DAI to **5,000 DAI**.
+- The `hop` will be set to **1,577 seconds**. The `hop` parameter defines the minimum amount of time that must elapse between `kick`s, therefore setting the maximum frequency of market actions.
+- The `want` will be set to **0.98**. The `want` parameter defines the relationship between the purchase price of MKR and the price provided by the MKR/USD oracle, i.e. the slippage. A want of 0.98 means that the purchase price of MKR can be up to 2% worse than the oracle price. However, if the price of MKR on Uniswap is better than the Oracle price, the Smart Burn Engine will not be constrained.
+- The `pip` will be set to `0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df`. The `pip` is the oracle referenced by the Smart Burn Engine.
+- The `receiver` will be set to the Pause Proxy address - `0xdbbe5e9b1daa91430cf0772fcebe53f6c6f137df `. The `receiver` is the address that LP tokens will be deposited to after each market action.
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:60]: Receiver address is incorrect - should be 0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB

Etherscan links for both pip and receiver address?

File: governance/votes/Executive vote - July 12, 2023.md [hunk 9]
```diff
@@ -0,0 +1,202 @@
...
+
+As there can only be one `Flapper` active in the Maker Protocol, the old `MCD_FLAP` used for buy-and-burn will be deactivated.
+
+Please review the [discussion thread](https://forum.makerdao.com/t/introduction-of-smart-burn-engine-and-initial-parameters/21201) for more information.
+
+### Keeper Job Updates
+
+As per this [post](https://forum.makerdao.com/t/dsscron-housekeeping-additions/21292) from the Protocol Scope Facilitators, the following addresses related to the Maker Keeper Network (DssCron) will be added to the [Chainlog](https://chainlog.makerdao.com/), if this executive proposal passes.
+
+- **Sequencer:** `0x238b4E35dAed6100C6162fAE4510261f88996EC9`
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:73]: In general, add Etherscan links or no?

File: governance/votes/Executive vote - July 12, 2023.md [hunk 10]
```diff
@@ -0,0 +1,202 @@
...
+|VoteWizard |[0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1](https://etherscan.io/address/0x9E72629dF4fcaA2c2F5813FbbDc55064345431b1) |1.6|
+|Navigator |[0x11406a9CC2e37425F15f920F494A51133ac93072](https://etherscan.io/address/0x11406a9CC2e37425F15f920F494A51133ac93072) |0.32|
+
+This is a total of **229.44 MKR**.
+
+### First Stage of CRVV1ETHSTETH-A Offboarding
+
+As per this [post](https://forum.makerdao.com/t/stability-scope-parameter-changes-3/21238) from Block Analitica, CRVV1ETHSTETH-A meets the criteria for offboarding detailed in the [Stability Scope](https://mips.makerdao.com/mips/details/MIP104). The following initial actions will be taken if this executive proposal passes.
+
+- Remove CRVV1ETHSTETH-A from the DC-IAM (autoline).
```
Comment (astronautthis) [inline governance/votes/Executive vote - July 12, 2023.md:135]:
```suggestion
- Remove CRVV1ETHSTETH-A from the [DC-IAM (autoline)](https://manual.makerdao.com/module-index/module-dciam).
```

File: governance/votes/Executive vote - July 12, 2023.md (hunk 2 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - July 12, 2023.md:42]: most of these are intentional - `Dai` is the name of a currency, `DAI` is a unit of measurement. This one is an error though

File: governance/votes/Executive vote - July 12, 2023.md (hunk 9 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - July 12, 2023.md:73]: typically haven't done this for these kind of entries, it's different to e.g. a payment address. See recent example here: https://github.com/makerdao/community/blob/master/governance/votes/Executive%20vote%20-%20June%2014%2C%202023.md#relevant-smart-contract-information

File: governance/votes/Executive vote - July 12, 2023.md (hunk 6 above)
Comment (patrick-j-govalpha) [reply governance/votes/Executive vote - July 12, 2023.md:2]: CRVV1ETHSTETH-A 1st Stage Offboarding falls under Scope Defined Parameter Changes for the purpose of the title. Agree on the Core Unit bit

**Pull request merged**